### PR TITLE
Add the structure to follow in a dockerfile.

### PR DESCRIPTION
**Pull request by fmontoto**
This pull request intend to create a style guide for writing docker files.

### CODE REVIEW

**Comment by camilog** [comment]
Here's a couple of useful links on how to write good Dockerfiles:
https://docs.docker.com/engine/reference/builder/ (reference)
http://crosbymichael.com/dockerfile-best-practices.html (best practices)

**Comment by fmontoto** [comment]
Links added

File: README.md [hunk 1]
```diff
@@ -4,3 +4,23 @@ Repositoy to store Dockerfiles for Niclabs projects
 # Contributing
 This repository is intended to contain only Dockerfiles and instructions related to its use.
 Place your Dockerfiles and related files at `/project_name/ ` 
+
+## How to do the Dockerfile?
+We are following some simple conventions to make the dockerfiles to look alike among them.
+### Structure
+1. Header:
+  1. *Fom*: You MUST have one docker from a specified version (FROM debian:**8**), you might use **latest** version but only if you have a fixed version of the same docker. This is intended to keep at least one version tested all the time.
```
Comment (camilog) [inline README.md]: Is "from"

File: README.md [hunk 2]
```diff
@@ -4,3 +4,23 @@ Repositoy to store Dockerfiles for Niclabs projects
...
+  2. *Maintainer*: Include a maintainer.
+2. [Dependencies]:
+  1. *Binary*: Install first the binary dependencies, and among them, start by the ones included in packages by the host OS. Use one line per dependency, this makes easier to see diff among versions.
```
Comment (Zeeerooo) [inline README.md:19]: I would add "In alphabetical order".

Also, I think is a good idea to show some examples (or reference one)

**Comment by fmontoto** [comment]
@Zeeerooo Added the alphabetical order. How about @camilog add the clean up section missing from [rata](https://github.com/niclabs/docker/blob/master/ratadns/fievel/Dockerfile) and we use it as sample

**Comment by camilog** [comment]
I prefer this Dockerfile as an example [ratadns-gopher](https://github.com/niclabs/docker/blob/master/ratadns/gopher/Dockerfile), maybe I need to add a few comments

**Comment by fmontoto** [comment]
Done.

File: README.md (hunk 1 above)
Comment (camilog) [inline README.md]: This hasn't been fixed yet

Comment (fmontoto) [inline README.md]: Fixed, I didn't see it sorry

**Comment by camilog** [comment]
LGTM

**Comment by Zeeerooo** [comment]
LGTM

**Pull request merged**